modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/definition/tree/manager/test/ObjectDefinitionTreeUtilTest.java

```diff
@@ -1081,6 +1083,81 @@ public void testBindPublishedObjectDefinitions() throws Exception {
 			_objectEntryLocalService, _objectRelationshipLocalService);
 	}
 
+	@Test
+	public void testDeleteNodeObjectEntry() throws Exception {
+
+		// Delete object entry
+
+		ObjectDefinition rootObjectDefinition1 =
```

Comment:
Can we follow the same pattern you used in other tests, like objectDefinitionA, objectDefinitionB, and objectDefinitionAA? Follow this nomeclature for the rest of the test